NUnitTestMarsRover/MoveBacwardCommandWestHeadingTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using NUnit.Framework;
```

Comment:
This test name is spelled wrong.

However, I think you may have taken splitting up these tests to the extreme and you lost the benefit of having a `[SetUp]` method...

For every production file you have, you should have one test file (in this case there appear to be 4).

So really you need a test file called `MoveBackwardCommandTests.cs` that should combine the 4 test files you have with the various headings.  This obviously applies to the other directions as well.  The naming pattern I used there is `[Name of Prodction File]Tests.cs`.

Reply:
Let me clarify...the benefit of a `[SetUp]` method is most valuable when you have many tests in a single file that share the same boilerplate code...whereas in this file you have 1 test so it's hard to see how a `[SetUp]` function is valuable.

Reply:
Ok, I may have misunderstood your previous comment about how to split these up.   Not a big deal to modify them.